app/schemas/models/api-client.schema.coffee

```diff
@@ -4,6 +4,10 @@ APIClientSchema = {
   description: 'Third parties who can make API calls, usually on behalf of a user.'
   type: 'object'
   properties: {
+    creator: {
+      type: 'object',
+      description: 'Id of user who created this APIClient'
+    }
```

Comment:
This is working fine, but just curious that why did we not use c.objectId for this like this: https://github.com/codecombat/codecombat/blob/master/app/schemas/models/prepaid.schema.coffee#L4

Reply:
I don't want the type to optionally be a string:
https://github.com/codecombat/codecombat/blob/oauth-creator/app/schemas/schemas.coffee#L25

I'm not entirely sure how the links property works, but the only reference I could find was in treema code here:
https://github.com/codecombat/codecombat/blob/oauth-creator/app/core/treema-ext.coffee